src/language/en-US.json

```diff
@@ -148,6 +148,7 @@
   "divisibility_can_not_more_than_6": "Divisibility cannot exceed 6.",
   "divisibility_invalid": "This divisibility is invalid [0-6]",
   "don_not_disclose": "Please, keep this information private. Sharing your mnemonic passphrase with others may put your funds at risk.",
+  "don_not_disclose_title": "Keep this private!",
```

Comment:
please fix the translation key typo. `do_not_disclosed_title`.

Reply:
I've copied the key from the line above, that's why we have that, I'll fix them both.

Reply:
Fixed